### PR TITLE
Help: Fix capitalization to standard rules

### PR DESCRIPTION
**Pull request by jwrober**
Fixes missed capitalized rules now that we have standardized on what to capitalize.

### CODE REVIEW

**Comment by lmoureaux** [comment]
Are you planning for more? Asking because the PR is still a draft

**Comment by jwrober** [comment]
> Are you planning for more? Asking because the PR is still a draft

No, forgot it was set as draft. Very few actual text changes in the PR. I used `sed` to do most of the changes.

File: data/helpdata.txt [hunk 1]
```diff
@@ -70,161 +70,161 @@ of the game.\
...
+"), _("\
+ 5. Deciding where to build cities.\
+"), _("\
+The best location for a city is a matter of taste. A city which is placed near the sea is easier to spot by \
+opponents, but can also serve as a port for seagoing units. They also usually need a Coastal Defense city \
+improvement later on for defense against an attacking navy. The best strategy is to build a few of both, but \
+keep in mind that your opponents will find it harder to locate your city if you do not build it by the sea. \
+Another thing to consider is the land around the city in its \"Vision Radius\". The vision radius is the the \
+map tiles that the city's population can manage as it grows. Many land tiles will have Resources on them \
+(called Resources) that give food, production, or trade bonuses. Use the middle-click on your mouse to see. \
```
Comment (lmoureaux) [inline data/helpdata.txt:139]: "Resources" is used in the generic sense here, so no capital

File: data/helpdata.txt [hunk 2]
```diff
@@ -351,176 +352,176 @@ categories = "Bonus", "Nuisance"
...
+initial revenue by entering the marketplace and selling trade goods, but it is reduced to a third.\
 "), _("\
-This limit is enforced only when attempting to establish a new Trade Route. If you temporarily lose the \
-ability to establish as many Trade Routes as a City already has, existing ones are not affected.\
+This limit is enforced only when attempting to establish a new trade route. If you temporarily lose the \
+ability to establish as many trade routes as a city already has, existing ones are not affected.\
 "), _("\
-Beware: in rulesets where Plague is enabled, it can travel along Trade Routes. See the section on Plague for \
-more details.\
+Beware: in rulesets where Plague is enabled, it can travel along trade routes. See the page on Plague in the \
```
Comment (lmoureaux) [inline data/helpdata.txt:513]: Minor: I'd write "the Plague page" or "the page on plague" (the first referring to the title, the second to the event)

**Pull request merged**